core/keyboard_nav/tab_navigate_cursor.js

```diff
@@ -14,35 +14,34 @@
 goog.module('Blockly.TabNavigateCursor');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.ASTNode');
-goog.require('Blockly.BasicCursor');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.Field');
+const ASTNode = goog.require('Blockly.ASTNode');
+const BasicCursor = goog.require('Blockly.BasicCursor');
+const Field = goog.requireType('Blockly.Field');
+const {inherits} = goog.require('Blockly.utils.object');
 
 
 /**
  * A cursor for navigating between tab navigable fields.
  * @constructor
- * @extends {Blockly.BasicCursor}
+ * @extends {BasicCursor}
  */
 const TabNavigateCursor = function() {
   TabNavigateCursor.superClass_.constructor.call(this);
 };
-Blockly.utils.object.inherits(TabNavigateCursor, Blockly.BasicCursor);
+inherits(TabNavigateCursor, BasicCursor);
 
 /**
  * Skip all nodes except for tab navigable fields.
- * @param {Blockly.ASTNode} node The AST node to check whether it is valid.
+ * @param {?ASTNode} node The AST node to check whether it is valid.
```

Comment:
I think yesterday at the team meeting we decided not to change these. But this one is clearly supposed to be `?` so all good :D

Reply:
Yeah, I had it included as part of my script, but I've removed it now and going forward I'm going to leave as is.